wrapper/src/main/java/software/amazon/jdbc/ConnectionPluginChainBuilder.java

```diff
@@ -82,15 +84,16 @@ public class ConnectionPluginChainBuilder {
         {
           put(DriverMetaDataConnectionPluginFactory.class, 100);
           put(DataCacheConnectionPluginFactory.class, 200);
-          put(AuroraHostListConnectionPluginFactory.class, 300);
-          put(AuroraConnectionTrackerPluginFactory.class, 400);
-          put(AuroraStaleDnsPluginFactory.class, 500);
-          put(ReadWriteSplittingPluginFactory.class, 600);
-          put(FailoverConnectionPluginFactory.class, 700);
-          put(HostMonitoringConnectionPluginFactory.class, 800);
-          put(IamAuthConnectionPluginFactory.class, 900);
-          put(AwsSecretsManagerConnectionPluginFactory.class, 1000);
-          put(LogQueryConnectionPluginFactory.class, 1100);
+          put(ReaderClusterConnectionPluginFactory.class, 300);
```

Comment:
NIT: what about sorting that alphabetically?

Reply:
I don't think the plugins in the builder here are meant to be sorted alphabetically.